code-challenges.rb

```diff
@@ -10,6 +10,17 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+# def is_this_odd (number)
+#     if number.even?
+#         p "#{number} is even"
+#     elsif number.odd?
+#         p "#{number} is odd"
+#     else
+#         p 'enter a number pls'
+#     end
+# end
+
+# is_this_odd(num3)
```

Comment:
Looks great. Love the else statement as a catchall. Just remember to uncomment your code before submitting.